tests/org/eolang/seq-tests.eo

```diff
@@ -25,8 +25,12 @@
 +home https://github.com/objectionary/eo
 +junit
 +package org.eolang
-+version 0.29.0
++version 0.29.1
 
+# @todo #1299:60min Implement test that will
```

Comment:
@volodya-lombrozo You forgot to remove this todo

Reply:
@Graur Thank you. Totally forgot about it. Done.